index.html

```diff
@@ -5202,24 +5219,32 @@ <h2 id="AudioWorkerParamDescriptor">
             source is pointing in 3D Cartesian coordinate space. Depending on
             how directional the sound is (controlled by the <b>cone</b>
             attributes), a sound pointing away from the listener can be very
-            quiet or completely silent. The default value is 1. This parameter
-            is a-rate.
+            quiet or completely silent. The default value is 1. The default
+            value is 0. This parameter is <a>a-rate</a> when <a
```

Comment:
Too much cut-n-paste; remove the part about default is 0 because the default is 1.